packages/nuxt/src/app/entry.ts

```diff
@@ -53,7 +53,11 @@ if (process.client) {
     import.meta.webpackHot.accept()
   }
 
+  // eslint-disable-next-line
+  let app: Promise<any>
```

Comment:
Shall we explicitly call it `nuxtApp`/`vueApp` to avoid future conflicts and naming consistency?